Gruntfile.js

```diff
@@ -134,10 +134,10 @@ module.exports = function(grunt) {
 
   var browsers = [{
     browserName: "firefox",
-    version: "30"
+    platform: "linux"
   }, {
     browserName: "chrome",
-    version: "35"
+    platform: "linux"
```

Comment:
Most of our users will probably be on Windows.  We probably want to align to that fact.